eng/sdl-tsa-vars.config

```diff
@@ -0,0 +1,14 @@
+-SourceToolsList @("policheck","credscan")
```

Comment:
So don't think this will do anything unless the following conditions are met:
- Arcade post build validation is enabled in the repo (it's not). **or**
- The repo shows up in the staging pipeline drop (it doesn't).

So, I think we should go with just the `tsaoptions.json` for now. It would be good at least to get some experience as to whether it works as we expects, files bugs. THat way we can move away from bespoke solutions and onto standard patterns over time.

Reply:
OK, I'll remove it. Want me to leave it in the other two PR's (ie just do the experiment in this repo)?

Reply:
Let's merge this PR, see the results, and then we can make a call on the other two